pkgs/build-support/vm/default.nix

```diff
@@ -23,7 +23,7 @@ rec {
   hd = "vda"; # either "sda" or "vda"
 
   initrdUtils = runCommand "initrd-utils"
-    { buildInputs = [ nukeReferences ];
+    { nativeBuildInputs = [ buildPackages.nukeReferences ];
```

Comment:
This is likely needed because `nukeReferences` does not come from `callPackage` context where our cross-compile overlay can override it.

Reply:
Exactly. The alternative would be to pass `nukeReferences` as an argument fulfilled one level up by `callPackage`. I think then the blanket `with pkgs;` should go away as it would overlay these names. Do you think this is the better solution?

In addition to `nukeReferences`, I believe the dependencies `perl`, `perlPackages.XMLSimple`, and `dpkg` further down would need the same treatment.